### ISSUE TITLE
[filterscripts] use I3Reader traysegment so the filestager works (Trac #1993)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1985]
Non-production users would like to use the stagers automatically.  This shouldn't affect production.

Migrated from https://code.icecube.wisc.edu/ticket/1993
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:57", 
    "description": "Non-production users would like to use the stagers automatically.  This shouldn't affect production.", 
    "reporter": "david.schultz", 
    "cc": "joertlin", 
    "resolution": "fixed", 
    "_ts": "1550067237750774", 
    "component": "combo reconstruction", 
    "summary": "[filterscripts] use I3Reader traysegment so the filestager works", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2017-05-01T17:47:56", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170522T18:10:27: 
* david.schultz commented:

> In changeset 155945:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="155945"
> add I3Reader tray segment instead of module.  allows automatic usage of filestagers. closes #1985
> ```


* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:57: 
* olivas commented:

> Milestone June 2017 deleted


* olivas changed milestone from "June 2017" to ""
